recipe/meta.yaml

```
@@ -15,6 +15,7 @@ source:
     - patches/0001-Enable-latest-libcxx-on-MacOS.patch                # [osx]
     - patches/0002-Remove-nvidia-nccl-cu12-from-pyproject.toml.patch
     - patches/0003-Mark-wheels-as-any-platform-compatible.patch
+    - patches/0004-Use-RMM-s-pached-CCCL.patch                        # [linux]
```

Typo: patched. Can we rename the file too?
```suggestion
    - patches/0004-Use-RMM-s-patched-CCCL.patch                       # [linux]
```

---

Oops 😂

Good catch Bradley. Have pushed a fix